src/dashboard/src/settings/components/oidc_auth.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+# AUTH_SERVER = 'https://login.microsoftonline.com/common/v2.0/'
```

Comment:
What's this line for?

Reply:
Nothing useful. :smile: I'll get rid of it.